pkg/cluster/cluster.go

```diff
@@ -479,6 +479,15 @@ func (c *Cluster) pollPods() ([]*v1.Pod, []*v1.Pod, error) {
 }
 
 func (c *Cluster) updateStatus() error {
+	if c.bm != nil {
+		bs, err := c.bm.getStatus()
```

Comment:
I would recommend in (*cluster).run():
```
c.reconcile(running)
... put here ...
c.updateStatus()
```
After reconcile(), before updateStatus(), setBackupStatus() there.
In the future, I would like to put update status routine into reconcile() to eliminate unpredictability of each reconcile time -- each reconcile would only took limited time.